util/mbed/mbed-copy-osx.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+import os
+import time
+import shutil
+import serial
+import argparse
+import mbed_lstools as mbedls
```

Comment:
yes, it probably will work as it stands now - haven't tested it on anything besides os x tho